index.html

```diff
@@ -69,6 +70,43 @@ <h4>Enterprise</h4>
           </div>
         </div>
       </section>
+
+      <section class="clients">
+        <div class="clients__testimonials">
+          <h4>Testimonials</h4>
+          <h2>What Clients Says About Us</h2>
+          <hr />
+          <p>
+            Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolore, non! Saepe mollitia, dolore pariatur hic laudantium
+            doloribus, soluta odit, molestias voluptas excepturi tempora. Architecto, obcaecati!
+          </p>
+          <p>
+            Lorem ipsum dolor sit amet consectetur adipisicing elit. Doloribus eligendi tenetur voluptatum, quas consequuntur
+            consectetur.
+          </p>
+          <div class="clients__arrows">
+            <span>&LeftArrow;</span>
+            <span>&RightArrow;</span>
+          </div>
+        </div>
+        <div class="clients__card">
+          <img src="" alt="" />
+          <q>Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam ducimus labore asperiores, vel ab ipsum!</q>
+          <div class="clients__card-bottom">
+            <i class="lni lni-quotation"></i>
+            <img src="" alt="" />
```

Comment:
img src & alt missing

Reply:
cannot find the photo in the img folder

Reply:
could you find an alternative somewhere? double check if it's not in the bxl-johnsons 1st (if not already done)

Reply:
ok, alternatively i use a rnd img

Reply:
i take a screenshot so i dont waste time on it